code/modules/mob/mob_helpers.dm

```diff
@@ -29,6 +29,12 @@
 			return 1
 	return 0
 
+/proc/check_holy()
+	var/turf/T = get_turf(src)
+	if(!T)	return 0
+	if(!T.holy)	return 0

```

Comment:
try to use spaces for non-syntactic alignment, many things (like github) have broken tab settings and won't display it properly with tabs